### PR TITLE
feat: added sticky nav bar

### PR DESCRIPTION
**Pull request by Srija-Burugula**
Closes: issue#774

Changes made: 
Added sticky nav bar. 

Screenshots:
Screenshot 2023-06-18 230439

### CODE REVIEW

**Comment by tamalCodes** [comment]
Hi @Srija-Burugula , always check whenever you type. 
Currently the issue number is not being highlighted 
image


This is because you added an unnecessary word `issue` , instead it should also look like this: `Closes: #774` 
image


Also the Pull request template is there for a reason, after making a PR - it should look like this👇🏻, but you modified the PR template which is not at all a good thing. Please fix the PR.
image

File: mil1.env
Comment (tamalCodes) [inline mil1.env:1]: Please remove this file. You are not supposed to add any `.env` to the commits.

**Review by tamalCodes — CHANGES_REQUESTED** [review]
The rest of the code looks good to me. Please make the suggested changes.

**Comment by Srija-Burugula** [comment]
I'm really sorry @tamalCodes. I'll make the changes and resubmit the PR.

**Comment by tamalCodes** [comment]
> I'm really sorry @tamalCodes. I'll make the changes and resubmit the PR.

Just do your changes in this PR itself, you don't need to make another one. 
Or if you want to make another one you can too - your call.

**Pull request closed without merging**